a/arrow-Go/arrow-go_ubi_8.5.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package          : arrow-Go
+# Version          : go/v10.0.1
+# Source repo      : https://github.com/apache/arrow
+# Tested on        : UBI 8.5
+# Language         : Go
+# Travis-Check     : True 
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Vinod.K <Vinod.K1@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=arrow
```

Comment:
@vinodk99 Lets make change to package name. All apache packages , we will add as `apache-<package>`.
So please make changes at all places in build-script and build_info.json